browser-app/package.json

```diff
@@ -27,10 +27,19 @@
     "@theia/cli": "next"
   },
   "scripts": {
-    "prepare": "theia build --mode development",
-    "start": "theia start",
-    "test": "true",
-    "watch": "theia build --watch --mode development"
+    "prepare": "yarn run clean && yarn build",
+    "clean": "theia clean && rimraf errorShots",
+    "build": "theiaext compile && theia build --mode development",
+    "watch": "concurrently -n compile,bundle \"theiaext watch --preserveWatchOutput\" \"theia build --watch --mode development\"",
+    "start": "theia start --plugins=local-dir:../../plugins",
+    "start:debug": "yarn start --log-level=debug",
+    "test": "wdio wdio.conf.js",
+    "test-non-headless": "wdio wdio-non-headless.conf.js",
+    "coverage:compile": "yarn build --config coverage-webpack.config.js",
+    "coverage:remap": "remap-istanbul -i coverage/coverage.json -o coverage/coverage-final.json --exclude 'frontend/index.js' && rimraf coverage/coverage.json",
+    "coverage:report:html": "istanbul report --root coverage --format html",
+    "coverage:report:lcov": "istanbul report --root coverage --format lcov",
+    "coverage": "yarn coverage:compile && yarn test && yarn coverage:remap && yarn coverage:report:lcov && yarn coverage:report:html"
```

Comment:
update: tested quickly this coverage script and it did not run. Since I do not think we need this ATM, should we just remove the coverage-related content?

Remove ?